.goreleaser.yml

```diff
@@ -57,8 +57,10 @@ builds:
     hooks:
       post:
         - cmd: make download-licenses
-        - cmd: bash scripts/gon_filepath_editor.sh {{ .Path }} amd64
-        - cmd: gon dist/gon_confluent_darwin_amd64.hcl
+        - cmd: 'codesign -s "Developer ID Application: Confluent, Inc." -v {{ .Path }} --options=runtime'
+          output: true
+        - cmd: zip "{{ .Path }}_signed.zip" {{ .Path }}
+        - cmd: xcrun notarytool submit "{{ .Path }}_signed.zip" --apple-id {{ .Env.AC_USERNAME }} --password {{ .Env.AC_PASSWORD }} --team-id RTSX8FNWR2 --wait
```

Comment:
Is the team-id something that we're good to hardcode here?

Reply:
Yep, it's similar to hardcoding an email, but we all share it (decided this would be better than asking you all to add yet another environment variable to your zshrc)